modules/WebServer.py

```diff
@@ -4,29 +4,47 @@
 server = None
 web_server_ip = "0.0.0.0"
 web_server_port = "8000"
+web_server_template = "www"
 
 
-def initialize_web_server(config):
-    import threading
-    global web_server_ip, web_server_port
-
-    if config.has_option('BOT', 'customWebServerAddress'):
-        custom_web_server_address = (config.get('BOT', 'customWebServerAddress').split(':'))
-        if len(custom_web_server_address) == 1:
-            custom_web_server_address.append("8000")
-            print "WARNING: Please specify a port for the webserver in the form IP:PORT, default port 8000 used."
+def get_web_server_setting(config, option, default):
```

Comment:
config.get already has a default value option this function is not required:
https://github.com/BitBotFactory/poloniexlendingbot/blob/master/modules/Configuration.py#L44